lib/git_reflow/workflows/core.rb

```diff
@@ -15,8 +15,9 @@ class Core
       command(:setup, defaults: {local: false, enterprise: false}) do |**params|
         reflow_options             = { project_only: params[:local], enterprise: params[:enterprise] }
         existing_git_include_paths = GitReflow::Config.get('include.path', all: true).split("\n")
+        evaluated_config_path = GitReflow::Config::CONFIG_FILE_PATH.gsub(/\$([a-zA-Z_]+[a-zA-Z0-9_]*)|\$\{(.+)\}/) { ENV[$1 || $2] }
```

Comment:
Love the regex matching here. Is there anyway to update (CONFIG_FILE_PATH)[https://github.com/reenhanced/gitreflow/blob/908b639b5cb0c861264c3cb3ec8fe70828721e6e/lib/git_reflow/config.rb#L5] to accurately reflect the correct path?  Maybe using `ENV['HOME']` inline instead of `$HOME` as it is?